src/clr-angular/layout/grid/_grid.clarity.scss

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('grid.clarity') {
+    .clr-row {
+        @include clr-make-row();
```

Comment:
I forgot to add a note here. We are using the new BS4 grid. We generate it the same way they do it. I am keeping it that way so that if we decide to upgrade later on, we know exactly which mixin to check and which file to check. Thats why the folder structure is also the same.